static_report/src/components/SingleReport/SRBarChart.tsx

```diff
@@ -13,17 +13,26 @@ export interface BarChartDatum {
 }
 interface Props {
   data: Array<BarChartDatum>;
+  height?: number | string;
+  xTicks?: number;
+  yTicks?: number;
 }
 
-export function SRBarChart({ data }: Props) {
+export function SRBarChart({ data, height = '230px', ...props }: Props) {
```

Comment:
do we still need this component and its code? 
![image](https://user-images.githubusercontent.com/11186480/186554149-93bdb8ff-3f35-4c5b-9e1b-584a051a36d7.png)

Same thing for `CRBarChart`